gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -119,6 +130,8 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     viewModel.getAddFeatureResults().observe(this, this::onFeatureAdded);
     viewModel.getUpdateFeatureResults().observe(this, this::onFeatureUpdated);
     viewModel.getDeleteFeatureResults().observe(this, this::onFeatureDeleted);
+    viewModel.getAddPolygonRequests()
```

Comment:
We are moving away from `Event.ifUnhandled`, it should probably be deprecated. Instead, for events that shouldn't be replayed on configuration changes, you can use an `Observable` backed by a `PublishSubject`.